pyccel/codegen/printing/pycode.py

```diff
@@ -114,12 +114,41 @@ def _print_FunctionDef(self, expr):
         return code
 
     def _print_Return(self, expr):
-        return 'return {}'.format(self._print(expr.expr))
+        code = ''
+        if expr.stmt:
+            code += self._print(expr.stmt)+'\n'
+        if expr.args:
```

Comment:
Shouldn't this check `expr.expr` not `expr.args`?

Reply:
Or maybe the if isn't needed? I don't think `bool(expr.args)` can be anything other than `true`

Reply:
maybe i should check the length?

Reply:
`bool([])` gives `False` so just `if expr.expr` should be enough